src/helperFunctions/updateTaskDetails.ts

```diff
@@ -0,0 +1,13 @@
+import fetch from '@/helperFunctions/fetch';
+
+const updateTaskDetails = async (editedDetails = {}, taskID = '') => {
+  const response = fetch({
+    url: `${process.env.NEXT_PUBLIC_BASE_URL}/tasks/${taskID}`,
```

Comment:
move this to constants

Reply:
Same reason as above

Reply:
https://github.com/Real-Dev-Squad/website-status/pull/341/files/651934177cd1ce9f5f39096ee1f5525865444318..682bc2071d4ecaa4124cf86780f11e0dc0643639#r1032746984